.jscsrc

```diff
@@ -1,5 +1,6 @@
 {
   "esnext": true,
+  "verbose": true,
```

Comment:
This prints out the rule name when there's an error

instead of `Newline before curly brace for block statement is disallowed at lib/jsdom/living/attributes.js :`

it's `disallowNewlineBeforeBlockStatements : Newline before curly brace for block statement is disallowed at lib/jsdom/living/attributes.js :`